src/contexts/consentRequestContext/index.jsx

```diff
@@ -44,7 +44,8 @@ function ConsentRequestProvider({
 
 ConsentRequestProvider.propTypes = {
   children: T.node.isRequired,
-  consentRequest: T.oneOfType([Object]),
+  // eslint-disable-next-line react/forbid-prop-types
+  consentRequest: T.object,
```

Comment:
Ideally this would be a `T.shape` with the properties required tested, though, tbh, this is a lot nicer handled in typescript, where you don't need runtime propTypes, because it happens via the language

Reply:
Yes - totally agree - would love to get to T.shape but maybe something for the future?

Reply:
@solid-akb oh, absolutely!